gems/pending/MiqVm/MiqVm.rb

```diff
@@ -300,7 +306,7 @@ def format(event)
     puts "\t#{rt.guestOS}"
     if rt.guestOS == "Linux"
       puts "\n\t\t*** /etc/fstab contents:"
-      rt.fileOpen("/etc/fstab", &:read).each_line do |fstl|
+      rt.fileOpen("/etc/fstab") { |fo| fo.read }.each_line do |fstl|
```

Comment:
Actually Rubocop *did* go the other way...see the Rubocop output above (MiqVm.rb Line 309)